Makefile

```diff
@@ -84,26 +87,29 @@ engine-down:
 	docker-compose -f ./tests/core/engine_adapter/docker-compose.yaml down
 
 fast-test:
-	pytest -n auto -m "fast"
+	pytest -n auto -m "fast and not cicdonly"
 
 slow-test:
+	pytest -n auto -m "(fast or slow) and not cicdonly"
+
+cicd-test:
 	pytest -n auto -m "fast or slow"
 
 core-fast-test:
-	pytest -n auto -m "fast and not web and not github and not dbt and not airflow and not jupyter"
+	pytest -n auto -m "fast and not web and not github and not dbt and not airflow and not jupyter and not cicdonly"
 
 core-slow-test:
-	pytest -n auto -m "(fast or slow) and not web and not github and not dbt and not airflow and not jupyter"
+	pytest -n auto -m "(fast or slow) and not web and not github and not dbt and not airflow and not jupyter and not cicdonly"
 
 airflow-fast-test:
-	pytest -n auto -m "fast and airflow"
+	pytest -n auto -m "fast and airflow and not cicdonly"
```

Comment:
do you have to add these everywhere?isn't it a subset of fast?

Reply:
It's a subset of dbt so can be removed from most of these. I added it everywhere to make it easier to add more in the future, but i guess these should be rare.

Reply:
but fast already includes cicdonly, so if you just added that to airflow, would that work?

Reply:
yes, i think the only place we need it is `fast-test`, `slow-test`, and `dbt-test`

since cicdonly isn't part of airflow, "fast and airflow" shouldn't need the additional cicdonly exclusion